api/ccapi/include/model.h

```diff
@@ -95,14 +125,36 @@ class Model {
 
   /**
    * @brief     save model and training parameters into file
+   * @todo      deprecate this
    */
   virtual void saveModel() = 0;
 
+  /**
+   * @brief  load model states and training parameters from a file
+   * @param path_prefix path_prefix to save the model, if full path is not
+   * given, it should be saved inside working directory
+   * @param option option to save parameters
+   */
+  virtual void saveModel(const std::string &path_prefix,
```

Comment:
this is minor:  i wonder if `save()` is enough? or `Model` prefix is needed? given this is applied on `model` object anyway

Reply:
I think it's a good point. Changed as suggested